product.php

```diff
@@ -3,6 +3,7 @@
   <head>
     <title>Cars | Product</title>
   </head>
-  <body> 
+  <body>
+Developer product details here  
```

Comment:
It should be "Designer product details here"

Reply:
Okay